src/repository/noteRelations.repository.ts

```diff
@@ -47,4 +47,22 @@ export default class NoteRelationsRepository {
   public async getParentNoteIdByNoteId(noteId: NoteInternalId): Promise<number | null> {
     return await this.storage.getParentNoteIdByNoteId(noteId);
   }
+
+  /**
+   * Delete note ralation containing noteId
+   *
+   * @param noteId - id of the current note
+   */
+  public async deleteNoteRelationByNoteId(noteId: NoteInternalId): Promise<boolean> {
+    return await this.storage.deleteNoteRelationByNoteId(noteId);
+  }
+
```

Comment:
can we return old naming if we are deleting relations. where noteId is parentId too?